service/service.go

```diff
@@ -196,9 +198,25 @@ func New(config Config) (*Service, error) {
 		}
 	}
 
-	var metricsCollector *collector.Collector
+	var legacyCollector *collector.Collector
 	{
 		c := collector.Config{
+			Helper: newHelper,
+			Logger: config.Logger,
+
+			InstallationName:      config.Viper.GetString(config.Flag.Service.Installation.Name),
```

Comment:
personal preference, having this wired through and validated

Reply:
I don't follow what you mean. Wired where? Validated where?

Reply:
As in, we pull it from the config where it is validated, like `config.Logger` for example

Reply:
It is validated where it is used. It is not, and should not where it is only passed through.